### PR TITLE
[raylib] De-vendor dependencies

### PR DESCRIPTION
**Pull request by Thomas1664**
**Describe the pull request**

- #### What does your PR fix?
  Raylib used vendored glfw, stb_image and cgltf.
  The vendored version of glfw is from GLFW master and introduces `GLFW_MOUSE_PASSTHROUGH` which is not present in our version of GLFW. See upstream issue https://github.com/raysan5/raylib/issues/2784

### CODE REVIEW

File: ports/raylib/portfile.cmake [hunk 1]
```diff
@@ -10,25 +10,17 @@ These can be installed on Ubuntu systems via sudo apt install libgl1-mesa-dev li
...
-    PATCHES ${patches}
+    PATCHES fix-linkGlfw.patch
 )
 
-string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" SHARED)
-string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" STATIC)
-
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
-        hidpi SUPPORT_HIGH_DPI
```
Comment (Thomas1664) [inline ports/raylib/portfile.cmake:31]: The option `SUPPORT_HIGH_DPI` doesn't exist anymore on upstream and it wasn't changed to a similar name.

File: ports/raylib/portfile.cmake [hunk 2]
```diff
@@ -42,9 +34,7 @@ vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         -DBUILD_EXAMPLES=OFF
-        -DSHARED=${SHARED}
-        -DSTATIC=${STATIC}
```
Comment (Thomas1664) [inline ports/raylib/portfile.cmake:46]: Removed by upstream: https://github.com/raysan5/raylib/commit/05dfbf3cd42e63f56d978cafec63f22fd7538e9f

**Comment by Thomas1664** [comment]
Depends on upstream changes because of Windows build failures caused by GLFW.

**Comment by MonicaLiu0311** [comment]
> Depends on upstream changes because of Windows build failures caused by GLFW.

Is it possible to link the corresponding upstream PR here? It is convenient for me to check the progress.

**Comment by Thomas1664** [comment]
> > Depends on upstream changes because of Windows build failures caused by GLFW.
> 
> Is it possible to link the corresponding upstream PR here? It is convenient for me to check the progress.

Upstream closed the issue because they are not willing to support glfw which is not build from the glfw master branch.

**Comment by MonicaLiu0311** [comment]
>We can't support this port on Windows until upstream wants to fix glfw compatibility on Windows because of duplicated symbol errors.

Close this PR. Please Ping us to reopen if work work can go ahead.

**Pull request closed without merging**